build.sbt

```diff
@@ -147,3 +148,6 @@ lazy val microservice = Project(appName, file("."))
     Resolver.jcenterRepo,
     Resolver.bintrayRepo("hmrclt", "maven")
   ))
+  .settings(
+    testOptions in Test += Tests.Argument("-oDF")
```

Comment:
why is this required ?

Reply:
we were trying to obtain more of the stack trace but Andy and I don't think this made any difference, so removing ...